doc/compute_lambda_grid.txt

```diff
@@ -63,94 +72,95 @@ different ways.  For example, the values can be output by the
 :line
 
 The formula used to calculate the mean free path (lambda) is given in
-"(Bird94)"_#Bird94 as equation 4.65:
+"(Bird94)"_#Bird94 as equation 4.77:
 
 :c,image(Eqs/lambda.jpg)
 
-This is an approximate mean free path for a multi-species mixture,
-suitable for estimating optimal grid cell sizes as explained above.
-It is a simplied version of formulas 4.76 and 4.77 from the same
-reference.
+The formula used to calculate the mean collision time (tau) is given in
+"(Bird94)"_#Bird94 as equation 1.38 combined with 4.75:
 
-Dref and Tref and omega are collision properties for a reference
-species in the flow.  The reference species is specified by the
-{species} argument.  It must be a species defined by the
-"species"_species.html command and listed in the file of per-species
-collision properties read in by the "collide"_collide.html command.
+:c,image(Eqs/tau.jpg)
 
-Specifically, Dref is the diameter of molecules of the species, Tref
+These are the exact mean free path and mean collision time for a
```

Comment:
I think it would be good to put a NOTE comment somewhere in this section of the doc page to indicate we have changed the formulas (as of what date) for how this command calculates the mean-free path.  So that users can see what was used before and what the newer, more accurate formula is.  The NOTE comment could possibly include the old formula.  Also a sentence or two as to why we switched to the new formulation.  And if possible what kinds of simulations for which the change would make a significant difference.

Reply:
Thanks @aborner1!

Reply:
Done